ZipBuilder/Sources/ZipBuilder/FileManager+Utils.swift

```diff
@@ -106,7 +106,8 @@ extension FileManager {
     if #available(OSX 10.12, *) {
       let formatter = ISO8601DateFormatter()
       formatter.formatOptions.insert(.withInternetDateTime)
-      return formatter.string(from: Date())
+      // Replace ":" with "-" so that bash file completion still works.
+      return formatter.string(from: Date()).replacingOccurrences(of: ":", with: "-")
```

Comment:
Would an explicit custom date format be more readable?

Reply:
Current version looks like:

```
$ ls -lt | head
total 12536
drwxr-xr-x  12 paulbeusterien  primarygroup      384 Mar  5 15:21 2020-03-05T23-17-50Z
-rw-r--r--   1 paulbeusterien  primarygroup  6416981 Mar  5 15:21 Frameworks.zip
drwxr-xr-x  11 paulbeusterien  primarygroup      352 Mar  5 14:21 2020-03-05T22-19-43Z
drwxr-xr-x   5 paulbeusterien  primarygroup      160 Mar  5 14:00 2020-03-05T21-59-56Z
drwxr-xr-x   5 paulbeusterien  primarygroup      160 Mar  5 13:52 2020-03-05T21-50-17Z
drwxr-xr-x  12 paulbeusterien  primarygroup      384 Mar  5 13:42 2020-03-05T21-39-47Z
drwxr-xr-x   5 paulbeusterien  primarygroup      160 Mar  5 09:54 2020-03-05T17:54:33Z
drwxr-xr-x   5 paulbeusterien  primarygroup      160 Mar  5 09:46 2020-03-05T17-46-47Z
drwxr-xr-x   5 paulbeusterien  primarygroup      160 Mar  5 09:36 2020-03-05T17-36-15Z
```
It sortable, readable, and works in bash completion. I'm find with this or any other format that meets those three requirements.

Reply:
I mean instead of replacing symbols in the formatted string, if it would be better to use something like this:
```
let formatter = DateFormatter
formatter.dateFormat = "YYYY-MM-dd'T'HH-mm-ss"
...
```

Reply:
I see. Much cleaner. Thanks!